tests/unit/test_install.py

```diff
@@ -1324,3 +1326,23 @@ def test_databricks_runtime_version_set(ws, mock_installation):
     with pytest.raises(SystemExit) as err:
         WorkspaceInstaller(prompts, mock_installation, ws, product_info, environ)
     assert err.value.args[0] == "WorkspaceInstaller is not supposed to be executed in Databricks Runtime"
+
+
+def test_check_inventory_database_exists(ws, mock_installation):
+    ws.config.inventory_database = "ucx_exists"
+    prompts = MockPrompts(
+        {
+            r".*Inventory Database stored in hive_metastore": "ucx_exists",
+            r".*": "",
+        }
+    )
+    installation = Installation(ws, "ucx", install_folder="/Applications/ucx")
+    install = WorkspaceInstaller(prompts, installation, ws, PRODUCT_INFO)
+    Installation.existing = create_autospec(Installation.existing)
```

Comment:
Here you override the type method, corrupting the state of other threads when tests run in parallel. Instead, you should pass "Installation.existing" as a callable and mock it out for tests